README.md

```diff
@@ -268,6 +268,35 @@ keep it.
 Use [proofit404/blacken](https://github.com/proofit404/blacken).
 
 
+### PyCharm
+
+(Adapted from [*Pylinting with PyCharm*](https://kirankoduru.github.io/python/pylint-with-pycharm.html))
+
+1. Install `black`.
+
+        $ pip install black
+
+2. Locate your `black` installation folder.
+
+        $ which black         # MacOS/Linux
+        /usr/local/bin/black  # this is just a possible output check yours
```

Comment:
I like the simple comment you used for the Windows version: `# possible location`.